_providers/i3.net.md

```diff
@@ -3,6 +3,17 @@ name: i3.net
 status: OK
 domains:
   - i3.net
+server:
+  - type: imap
+    socket: STARTTLS
+    hostname: imap.i3.net
+    port: 143
+    username: EMAILADDRESS
```

Comment:
isn't this and also the smtp setting tried automatically?  It's probably still ok to merge this.  But note that they mailed me that they are soon adding the autoconfig.xml file -- maybe we can do a "Check" script for all entries in the DB to see if any added autoconfig.xml files meanwhile.  It's what we recommend because it prevents outdted entries.

Reply:
It does check those servers, but if I try to login without any more specifications it takes minutes.

But if the autoconfig values follow soon it should be fine of course. Let's close this next week if they added autoconfig until then.